tasks/redhat.yml

```diff
@@ -14,16 +14,30 @@
 # will fail. Remove these files if the interface does not exist or is not being
 # configured.
 
-- name: RedHat | remove invalid interface configuration
+- name: RedHat | remove invalid interface configuration (network-scripts)
   become: true
   file:
     path: "/etc/sysconfig/network-scripts/ifcfg-{{ item }}"
     state: absent
   when:
+    - not interfaces_use_networkmanager
```

Comment:
Should this be interfaces_use_nmconnection?

Reply:
The idea was to keep this task doing the same thing as before when using network-scripts on c8s.

Reply:
Both variables are set to the same value by default:

```
interfaces_use_networkmanager: "{{ ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version | int >= 9 }}"
interfaces_use_nmconnection: "{{ ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version | int >= 9 }}"
```

Reply:
The difference is that when interfaces_use_networkmanager is true but interfaces_use_nmconnection is false, NM is used in legacy mode and reads the ifcfg files.

Reply:
indeed, eb4x just stated the same, some people might use network manager with old script files - https://github.com/michaelrigart/ansible-role-interfaces/pull/136/commits/533f175566eaea5db941406897dedd067d53dc0e

Reply:
I have updated this check in #4.